scripts/jest/shouldIgnoreConsoleError.js

```diff
@@ -7,12 +7,8 @@ module.exports = function shouldIgnoreConsoleError(
 ) {
```

Comment:
We need to get this file down to zero because it's covering up a bunch of quirks.